docs/LINUX.md

```diff
@@ -101,25 +32,58 @@ enabled=1
 gpgcheck=1
 gpgkey=https://assets.logdna.com/logdna.gpg" | sudo tee /etc/yum.repos.d/logdna.repo
 
-# Update the package
-sudo yum update logdna-agent
-```
-
-The agent will uninstall the previous version and reuse the existing configuration file, by default
-located in `/etc/logdna.conf`.
-
-If you defined a `logdna-agent.conf` configuration file on a different location, and want to use it instead, you
-can specify it on your systemd unit file:
-
-```unit file (systemd)
-[Service]
-Environment="LOGDNA_CONFIG_FILE=/your/path/to/logdna.conf"
-```
-
-"Next, restart the agent:
+# install the package via yum (skip this step if already installed)
+sudo yum -y install logdna-agent
 
-```shell script
-sudo systemctl restart logdna-agent
+# update the package via yum
+sudo yum update logdna-agent
 ```
-
-[ingestion-key]: https://docs.logdna.com/docs/ingestion-key
+#
+## Usage
+The agent uses [systemd](https://systemd.io/) to run as a Linux daemon. The installed package provides a `systemd` unit file for the `logdna-agent` service which is defined to execute a daemon process with the compiled agent binary. The process started in the service is managed by `systemd` and can be interfaced with using `systemctl`.
+
+---
+### _Enabling the agent_
+- Reload the `systemd` unit files to ensure the most recent version of the agent is used & enable the service via `systemctl`:
```

Comment:
Instead of replacing the ampersand & with "and to", I moved the latter part into its own bullet since I think there's a slight distinction to be made. Reloading the files isn't a requirement in enabling the service, they are 2 separate commands so I broke them up accordingly.